includes/class-wc-cart-session.php

```diff
@@ -130,8 +130,17 @@ public function get_cart_from_session() {
 
 			} elseif ( ! $product->is_purchasable() ) {
 				$update_cart_session = true;
-				/* translators: %s: product name */
-				wc_add_notice( sprintf( __( '%s has been removed from your cart because it can no longer be purchased. Please contact us if you need assistance.', 'woocommerce' ), $product->get_name() ), 'error' );
+				/* translators: %1$s: product name. %2$s reason why */
```

Comment:
This comment is incorrect, need to restore `/* translators: %s: product name */`